app.rb

```diff
@@ -47,7 +47,7 @@ def openid_configuration
 
   def token(code)
     json HTTParty.post(
-      openid_configuration[:token_endpoint],
+      ENV['IDP_TOKEN_URL'] || openid_configuration[:token_endpoint],
```

Comment:
Well not back out the HTTParty change, I miss-spoke.

I think we should just provide `basic_auth` config directly when we GET or POST each endpoint


http://stackoverflow.com/a/7627913